data/sea/52-psv.h

```diff
@@ -163,8 +165,12 @@ static ierror_msg_t psv_read_buffer (psv_state_t *s)
 
         if (new_entity < 0) {
             if (entity_cur_size != 0) {
-                error = psv_write_outputs (s->output_fd, entity_cur, entity_cur_size, s->fleet);
-                if (error) goto on_error;
+                ierror_msg_t msg = psv_write_outputs (s->output_fd, entity_cur, entity_cur_size, s->fleet);
+                //error = psv_write_outputs (s->output_fd, entity_cur, entity_cur_size, s->fleet);
```

Comment:
yeah you're right, actually i probably should overhaul the `ierror_msg_t` vs `ierror_loc_t` stuff before I merge this